src/libraries/System.Diagnostics.FileVersionInfo/src/System/Diagnostics/FileVersionInfo.Unix.cs

```diff
@@ -31,13 +31,12 @@ private FileVersionInfo(string fileName)
             // that this should match for all intents and purposes.  If this ever becomes a problem,
             // we can implement a full-fledged Win32 resource parser; that would also enable support
             // for native Win32 PE files on Unix, but that should also be an extremely rare case.
-            if (!TryLoadManagedAssemblyMetadata())
-            {
-                // We could try to parse Executable and Linkable Format (ELF) files, but at present
-                // for executables they don't store version information, which is typically just
-                // available in the filename itself.  For now, we won't do anything special, but
-                // we can add more cases here as we find need and opportunity.
-            }
+            TryLoadManagedAssemblyMetadata();
```

Comment:
nit: Make it explicit that we are ignoring the return value
```suggestion
            _ = TryLoadManagedAssemblyMetadata();
```